src/main/java/uk/gov/hmcts/reform/sscs/config/NotificationConfig.java

```diff
@@ -10,9 +10,7 @@
 import org.springframework.beans.factory.annotation.Value;
 import org.springframework.core.env.Environment;
 import org.springframework.stereotype.Component;
-import uk.gov.hmcts.reform.sscs.ccd.domain.Benefit;
-import uk.gov.hmcts.reform.sscs.ccd.domain.LanguagePreference;
-import uk.gov.hmcts.reform.sscs.ccd.domain.State;
+import uk.gov.hmcts.reform.sscs.ccd.domain.*;
```

Comment:
I don't mind personally, but told we try not to use wildcards for imports.

Reply:
I saw that as well but im pretty sure I checked and with the changes ive added, I used all the classes within that package.